unime/src-tauri/capabilities/main.json

```diff
@@ -4,6 +4,7 @@
   "windows": ["main"],
   "permissions": [
     "core:default",
+    "deep-link:allow-get-current",
```

Comment:
We can use `default` here, since it also only contains `allow-get-current` ([see here](https://tauri.app/plugin/deep-linking/#default-permission)).